src/dmd/dmodule.d

```diff
@@ -553,12 +555,7 @@ extern (C++) final class Module : Package
         {
             if (!m.isRoot() && onImport)
             {
-                static __gshared bool bodge = false;
-                if(bodge)
-                    fprintf(global.stdmsg, "onImport\n");
                 auto onImportResult = onImport(m);
-                if(bodge)
-                    fprintf(global.stdmsg, "onImport %d\n", onImportResult);
```

Comment:
:+1:

Reply:
I wasn't sure about this.  Will wait for the testsuite to give an answer...